rule/var_naming.go

```diff
@@ -42,12 +44,15 @@ type VarNamingRule struct {
 	allowUpperCaseConst   bool                // if true - allows to use UPPER_SOME_NAMES for constants
 	skipPackageNameChecks bool                // check for meaningless and user-defined bad package names
 	extraBadPackageNames  map[string]struct{} // inactive if skipPackageNameChecks is false
+	pkgNameAlreadyChecked set                 // set of packages names already checked
 }
 
 // Configure validates the rule configuration, and configures the rule accordingly.
 //
 // Configuration implements the [lint.ConfigurableRule] interface.
 func (r *VarNamingRule) Configure(arguments lint.Arguments) error {
+	r.pkgNameAlreadyChecked = set{elements: map[string]struct{}{}}
```

Comment:
`Configure` method is usually used for rule's configurations, not for initialization. Could we avoid this line and move set's init to some other place?

Reply:
What is rule configuration other than initializing rule's fields?
Anyway, I'm okay for moving it. Suggestions are welcome.